packages/form-js-editor/src/render/components/palette/icons/index.js

```diff
@@ -11,6 +12,7 @@ import TextfieldIcon from './Textfield.svg';
 export const iconsByType = {
   button: ButtonIcon,
   checkbox: CheckboxIcon,
+  fieldset: Fieldset,
```

Comment:
Renaming to `FieldsetIcon` again, also this list is alphabetically sorted. Little nitpick but we should keep this that way.